setup.py

```diff
@@ -116,5 +116,6 @@ def get_requirements(requirements_file):
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
```

Comment:
Nit: You should remove the 3.5 classifier above.

Reply:
@feanil i'll fold this into another PR i just opened